### PR TITLE
Add a script to group by season and team for advanced stats

### PR DESCRIPTION
**Pull request by Wal8800**
### Summary

Calculating Offensive rating and Defensive Rating per season for each team (ie excluding the march madness games)

Feels free to double check the calculation since it might be wrong :laughing: 

I tried checking if the values are matching to other online source but it seem they include march madness games when calculating the possessions and rating

Previous output (calculating pos per game)
```
   Season  WTeamID  WScore  LScore     ORating     DRating
0    2003     1102    1603    1596  105.798118  105.336118
1    2003     1103    2127    2110  113.513101  112.605850
2    2003     1104    1940    1820  105.840624   99.293781
3    2003     1105    1866    1993   95.221207  101.701965
4    2003     1106    1781    1785   95.496735   95.711214
```

Current output (the advanced stats is all the way at the end)
```
                WScore  LTeamID  LScore  WFGM  WFGA  WFGM3  WFGA3  WFTM  WFTA  WOR  WDR  WAst  WTO  WStl  WBlk  WPF  LFGM  LFGA  LFGM3  LFGA3  LFTM  LFTA  LOR  LDR  LAst  LTO  LStl  LBlk  LPF          Pos     ORating     DRating
Season WTeamID                                                                                                                                                                                                                      
2003   1102       1603    36930    1596   536  1114    219    583   312   479  117  471   364  320   167    50  525   540  1188    133    348   383   539  269  564   256  363   152    44  514  1516.949514  105.672601  105.211148
       1103       2127    33977    2110   733  1508    147    434   514   698  264  538   411  341   196    63  536   750  1539    180    496   430   598  325  595   418  414   173    77  606  1873.805923  113.512289  112.605045
       1104       1940    35914    1820   673  1601    178    556   416   586  380  670   339  372   185   106  505   651  1554    178    536   340   480  305  634   327  388   155    89  539  1833.517252  105.807567   99.262769
       1105       1866    33024    1993   634  1602    197    540   401   568  351  601   378  485   242    54  526   702  1533    163    456   426   637  343  686   411  489   244   109  496  1958.671026   95.268678  101.752667
       1106       1781    34867    1785   656  1548    171    494   298   461  344  668   327  477   234    88  509   608  1495    134    426   435   615  317  626   330  422   246    89  452  1864.237504   95.535038   95.749603
```

### CODE REVIEW

**Review by changkon — COMMENTED** [review]
Is it correct to calculate possessions per game instead of calculating the possessions after the season?

File: advanced_stats_starter.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+pd.set_option('display.width', 1000)
+
+submission = pd.read_csv("input/SampleSubmissionStage1.csv")
+playoff_results = pd.read_csv("input/NCAATourneyDetailedResults.csv")
+seasons_results = pd.read_csv("input/RegularSeasonDetailedResults.csv")
+
+
+# https://www.basketball-reference.com/about/glossary.html
+def get_pos(row):
+    w_half = row.WFGA + 0.4*row.WFTA - 1.07*(row.WOR / (row.WOR + row.LOR))*(row.WFGA - row.WFGM) + row.WTO
```
Comment (changkon) [inline advanced_stats_starter.py]: Should be against opponent Defensive rebounding

**Comment by Wal8800** [comment]
how would I calculate possessions after the seasons ? would it be using the sum of those stats instead of those stats per game ?

**Comment by Wal8800** [comment]
how would I calculate possessions after the seasons ? would it be using the sum of those stats instead of those stats per game ?

**Comment by Wal8800** [comment]
calculating pos after the season cause the values to be bit different max most +- 1 but the order of the rating (ie team who have higher rating seem to be the same)

**Review by changkon — APPROVED** [review]

**Pull request merged**